app/vmauth/main.go

```diff
@@ -113,18 +113,23 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 		return true
 	}
 
+	// Limit per user concurrency of requests to backends
+	// This limit enables only if max_concurrent_requests value bigger that
+	// flag --maxConcurrentRequests value
+	if *maxConcurrentRequests < ui.MaxConcurrentRequests {
```

Comment:
I think, it's better to use if ui.MaxConcurrentRequests> 0 check.

 Otherwise per user limit works weird. In case when maxConcurrentRequests > ui.MaxConcurrentRequests && ui.MaxConcurrentRequests > 0 ui limit is ignored.